Sources/SKCore/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ add_library(SKCore STATIC
   XCToolchainPlist.swift)
 set_target_properties(SKCore PROPERTIES
   INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_Swift_MODULE_DIRECTORY})
-target_link_libraries(SKCore PRIVATE
+target_link_libraries(SKCore PUBLIC
```

Comment:
What's this for? Is sourcekit-lsp missing libraries that it uses in its target_link_libraries?

Reply:
Yes, we get issues about missing missing `Crypto.swiftmodule` (or something along those lines)